video-app/src/Pages/SinglePage/SinglePage.jsx

```diff
@@ -1,16 +1,29 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { Aside, SingleVideoCard } from "../../Component";
 import Navbar from "../../Component/Navbar/Navbar";
 import "./SinglePage.css";
 import { useParams } from "react-router-dom";
-import { SingleVideo } from "../../Utils/SingleVideo";
 import { useVideo } from "../../Context/VideoContext";
+import axios from "axios";
 
 const SinglePage = () => {
   const { Id } = useParams();
   const{state} = useVideo();
-  const {videos,error} = state;
-  const video = SingleVideo(videos, Id);
+  const {error} = state;
+  const [video,setVideo] = useState(null)
+
+  
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get(`/api/video/${Id}`)
+       setVideo(response.data.video)
+      } catch (error) {
+        console.log(error.response)
+      }
+    })()
+  },[Id])
```

Comment:
you can more specific to name these id for better readability

Reply:
will do that